experimental/stats/metricregistry.go

```diff
@@ -0,0 +1,240 @@
+/*
+ *
+ * Copyright 2024 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package stats
+
+import (
+	"log"
+
+	"google.golang.org/grpc/stats"
+)
+
+// DefaultMetrics are the default metrics registered through global metrics
+// registry. This is written to at initialization time only, and is read only
+// after initialization.
+var DefaultMetrics = stats.NewMetrics()
+
+// MetricDescriptor is the data for a registered metric.
+type MetricDescriptor struct {
+	// Name is the name of this metric. This name must be unique across whole
```

Comment:
Docstrings on fields don't follow this convention like types/functions.  You can just say "The name of the metric".  Throughout.  Maybe that's why it seems so redundant to me "Name is the name".  Just say "The name".

https://google.github.io/styleguide/go/decisions#doc-comments

Reply:
Maybe I'll just delete these comments.

Reply:
Actually switched comments for field to // The Field...